test/integration/apiserver/admissionwebhook/load_balance_test.go

```diff
@@ -40,6 +40,8 @@ import (
 	"k8s.io/client-go/rest"
 	kubeapiservertesting "k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
 	"k8s.io/kubernetes/test/integration/framework"
+
+	"github.com/foxcpp/go-mockdns"
```

Comment:
please drop this dependency and use the approach suggested in https://github.com/kubernetes/kubernetes/pull/122558#discussion_r1439979900